lib/cylc/scheduler.py

```diff
@@ -309,6 +336,21 @@ def _print_blurb():
             print logo_lines[i], ('{0: ^%s}' % lmax).format(license_lines[i])
         print
 
+    def _configure_pyro(self):
+        """Create and configure Pyro daemon."""
+        self.pyro = PyroDaemon(self.suite, self.suite_dir)
+        self.port = self.pyro.get_port()
+        port_file_path = self._check_port_file_exists(self.suite)
```

Comment:
Is this to guard against another suite instance starting up in between the first port file check and configuring pyro?